entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/mapper/ViewMapper.java

```diff
@@ -185,9 +187,15 @@ private ObjectMapper createAccessor(ManagedViewType<S> sourceType, ManagedViewTy
         }
 
         // Try to find a source attribute
-        MethodAttribute<? super S, ?> sourceAttribute = sourceType.getAttribute(targetAttribute.getName());
+        MethodAttribute<?, ?> sourceAttribute = (sourceType == null) ? null : sourceType.getAttribute(targetAttribute.getName());
+        AttributeAccessor accessor = (sourceAttribute == null)
+                ? Accessors.forEntityMapping((EntityViewManagerImpl) entityViewManager, targetAttribute)
+                : Accessors.forViewAttribute(null, sourceAttribute, true);
+        if (accessor == null) {
+            return null;
+        }
 
-        if (sourceAttribute == null) {
+        if ((sourceType != null) && (sourceAttribute == null)) {
```

Comment:
Returning null when the accessor is null is not a good idea I think. Maybe something along these lines is more appropriate.

```suggestion
        MethodAttribute<?, ?> sourceAttribute;
        AttributeAccessor accessor;
        if (sourceType == null) {
            sourceAttribute = null;
            accessor = Accessors.forEntityMapping((EntityViewManagerImpl) entityViewManager, targetAttribute);
        } else {
            sourceAttribute = sourceType.getAttribute(targetAttribute.getName());
            if (sourceAttribute == null) {
                if (targetAttribute.getMappingType() == Attribute.MappingType.PARAMETER) {
                    return new ParameterObjectMapper(((MappingAttribute<?, ?>) targetAttribute).getMapping());
                }
                // Optionally ignore missing attributes
                if (ignoreMissing) {
                    return null;
                }
                throw inconvertible("Attribute '" + targetAttribute.getName() + "' from target type is missing in source type!", sourceType, targetType);
            }
            accessor = Accessors.forViewAttribute(null, sourceAttribute, true);
        }
```